DEV-README.md

```diff
@@ -1,85 +1,169 @@
 # sos4R developer documentation
 
-## Package structure
+This file contains information for developers of the [R](http://r-project.org/) package `sos4R`.
+Documentation for usres can be found in the package's vignette (see `browseVignettes("sos4R")`).
 
-sos4R follows the regular R extension package structure. General documentation about R package development can be found at the following two websites.
+<!-- part of this page previously published at https://wiki.52north.org/Geostatistics/Sos4R -->
```

Comment:
Better provide a speaking link with some explanation like: 
```suggestion
[Ancient documentation](https://wiki.52north.org/Geostatistics/Sos4R) is available and should be considered if something is missing in this readme or before suggesting changes.
```

Reply:
Please the the wiki page - it is empty, though of course users could check the wiki history. I think that is unlikely.